_data/metadata.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	title: "Stuy Linux",
+	url: "https://stuylinux.org/",
+	language: "en",
+	description: "Stuyvesant Linux Club description",
```

Comment:
Maybe we could copy the tagline from our [StuyActivities page](https://stuyactivities.org/linux), or our [homepage](https://stuylinux.org/) for consistency

Reply:
agreed